### ISSUE TITLE
Feature request: Recognize windows-1252 coding with jschardet

### ISSUE THREAD

**Issue by Micha-he** [Acode-Foundation/Acode#787]
Maybe you can include the recognition of windows-1252 encoding from this js-version of chardet?

[https://github.com/aadsm/jschardet](https://github.com/aadsm/jschardet),

because ace is not handled the encoding themself, if you can read [here](https://github.com/ajaxorg/ace/issues/5230#issuecomment-1649497296)

**Comment by Micha-he** [comment]
Issues are closed, but still exists.

In the meantime I switched to QuickEdit+, where files with Windows-1252-coding works perfectly.

**Comment by bajrangCoder** [comment]
but `windowa-1252` encoding is already available in acode

**Comment by Micha-he** [comment]
Yes, but Acode opens the ansi-Windows-1252-files always as UTF-8!
I've added a demo-file below...

[acode-ansi-test.txt](https://github.com/user-attachments/files/20859592/acode-ansi-test.txt)

**Comment by Micha-he** [comment]
Great, works with the last beta-version.
